modules/migration-adding-cluster-to-cam.adoc

```diff
@@ -59,8 +68,31 @@ $ oc create route passthrough --service=image-registry --port=5000 -n openshift-
 ----
 
 * *Azure cluster*: You must select this option if you use Azure snapshots to copy your data.
-* *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
-* *Require SSL verification*: Optional: Select this option to verify SSL connections to the cluster.
+** *Azure resource group*: This field is displayed if *Azure cluster* is selected. Specify the Azure resource group.
++
+When an OpenShift cluster is created on Microsoft Azure, an Azure Resource Group is created to contain all resources associated with the cluster. You can display all resource groups by issuing `az group list` in the Azure CLI.
++
+`ResourceGroups` associated with OpenShift clusters are tagged, so where `sample-rg-name` is the value you would extract and supply to the UI:
++
+[source,terminal]
+----
+{
+  "id": "/subscriptions/...//resourceGroups/sample-rg-name",
+  "location": "centralus",
+  "name": "...",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": {
+    "kubernetes.io_cluster.sample-ld57c": "owned",
+    "openshift_creationDate": "2019-10-25T23:28:57.988208+00:00"
+  },
+  "type": "Microsoft.Resources/resourceGroups"
+},
+----
++
+This information is also available from the [Azure Portal](https://portal.azure.com) in the **Resource groups** blade.
```

Comment:
The link to the Azure portal doesn't render correctly in the preview, here's what it should be changed to:
```suggestion
This information is also available from the link:https://portal.azure.com[Azure Portal] in the **Resource groups** blade.
```

Source: [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites)